app-android/src/main/java/com/ustadmobile/port/android/view/ClazzEditFragment.kt

```diff
@@ -284,4 +317,376 @@ class ClazzEditFragment() : UstadEditFragment<ClazzWithHolidayCalendarAndSchoolA
     }
 
 
+}
+
+@OptIn(ExperimentalFoundationApi::class, ExperimentalMaterialApi::class)
+@Composable
+private fun ClazzEditScreen(
+    uiState: ClazzEditUiState = ClazzEditUiState(),
+    onClazzChanged: (ClazzWithHolidayCalendarAndSchoolAndTerminology?) -> Unit = {},
+    onClickSchool: () -> Unit = {},
+    onClickTimezone: () -> Unit = {},
+    onClickAddCourseBlock: () -> Unit = {},
+    onClickAddSchedule: () -> Unit = {},
+    onClickEditSchedule: (Schedule) -> Unit = {},
+    onClickDeleteSchedule: (Schedule) -> Unit = {},
+    onClickHolidayCalendar: () -> Unit = {},
+    onCheckedAttendance: (Boolean) -> Unit = {},
+    onClickTerminology: () -> Unit = {},
+    onClickHideBlockPopupMenu: (CourseBlockWithEntity?) -> Unit = {},
+    onClickIndentBlockPopupMenu: (CourseBlockWithEntity?) -> Unit = {},
+    onClickUnIndentBlockPopupMenu: (CourseBlockWithEntity?) -> Unit = {},
+    onClickDeleteBlockPopupMenu: (CourseBlockWithEntity?) -> Unit = {},
+) {
+    LazyColumn(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
```

Comment:
Do we need the Spacer items when we have the padding? Doesn't padding already solve the problem for us?